src/drive/mobile/modules/mediaBackup/duck/tmp.js

```diff
@@ -0,0 +1,63 @@
+/* global cozy */
+/* 
+This file is TEMPORARY. It covers a stack bug and we need to synchronize again some files
+
+!!TO BE DELETED WHEN datas are OK
+
+*/
+import { updateLibraryItem } from 'drive/mobile/lib/media'
+import { CURRENT_UPLOAD_PROGRESS } from './reducer'
+import throttle from 'lodash/throttle'
+import localforage from 'localforage'
+
+export const checkCorruptedFiles = async (
+  photosOnDevice,
+  alreadyUploaded,
+  client,
+  dispatch
+) => {
+  const onProgressUpdate = progress => {
+    dispatch({ type: CURRENT_UPLOAD_PROGRESS, progress })
+  }
+  const onThumbnailGenerated = thumbnailUrl => {}
```

Comment:
We could declare these functions outside of `checkCorruptedFiles ` so they aren't recreated with every call.

Reply:
`checkCorruptedFiles` is only call when there is a sync. Declaring `onProgressUpdate` outside means passing dispatch outside... Can do it but does it worth it? :D

Reply:
Oh right, I missed that `dispatch` is in the context. Probably not worth the bother then.